lib/components/easy-search-components.js

```diff
@@ -633,6 +633,42 @@ EasySearch.getComponentInstance = (function () {
         return autorun(function () {
           eventFunction(SearchVars.get(componentScope.generateId(), key));
         });
+      },
+      resetAutosuggest : function (context) {
```

Comment:
I would rather add an optional input as the parameter, to let the user find the input themselves. The way you are finding the input right now is somewhat hacky and with the input as an argument it would shorten the code needed to reset the autosuggest a lot.